### DISCUSSION TITLE
mounting directory from host into emby lxc

### DISCUSSION THREAD

**Discussion by collateral127** [tteck/Proxmox#3595] [Help Needed]
hi guys, i'm looking for a bit of help getting a directory from the host machine with media mounted directly to the lxc - i've currently got it mounted via fstab, but i was looking for a way to do it more directly, and if i'm completely honest, i'm a bit bewildered reading about how to do it online.

the sentiment i see is that the mp0/mp1 mount is the better way to do it, but i don't know how to do it in a way that makes sure to preserve the existing permissions on the directory, and the subuid/subgid thing confuses the hell outta me - if someone could help me get my head around it all, i'd be greatly appreciative :)

**Comment by chenks** [comment]
depends on your LXC - is it privelidged or unprivelidged?

if it's the latter than you can only do it the way you're alrady doing it - mount it in pve then bind mount via the conf file
if it's privelidged then you can mount the share direct in the LXC without first mounting it in pve.

for unprivelidged, if you mount it in fstab in the pve (with appropraite permissions) then once you bind mount via the conf you just need to ensure you set the main permissions within the LXC mount directory.

take a look at this for making sure permissions are set correctly
https://forum.proxmox.com/threads/tutorial-unprivileged-lxcs-mount-cifs-shares.101795/

**Comment by collateral127** [comment]
it's running privileged at the moment, and i think most of the tutorials i've seen talk about unprivileged, which i think has been the cause of my confusion - so i should be able to directly mount it, as the privileged lxc is able to see the entire underlying filesystem?

**Reply by chenks** [reply]
> it's running privileged at the moment, and i think most of the tutorials i've seen talk about unprivileged, which i think has been the cause of my confusion - so i should be able to directly mount it, as the privileged lxc is able to see the entire underlying filesystem? 

Yes
